extension/content.css

```diff
@@ -181,3 +181,18 @@
 #refined-github-readme-edit-link:hover {
 	opacity: 1;
 }
+
+/* style for minimize/maximize button group */
+.btn-group.refined-github-btn-group {
+    margin-right: 25px;
+}
+.file-header {
+    cursor: zoom-out;
+}
+.refined-github-minimized .file-header {
+    cursor: zoom-in;
```

Comment:
The cursor doesn't show consistently when I hover over the header. Only when I hover over the absolute top of the header.